linera-rpc/tests/format.rs

```diff
@@ -27,9 +28,20 @@ fn get_registry() -> Result<Registry> {
             .record_samples_for_newtype_structs(true)
             .record_samples_for_tuple_structs(true),
     );
-    let samples = Samples::new();
+    let mut samples = Samples::new();
     // 1. Record samples for types with custom deserializers.
     // 2. Trace the main entry point(s) + every enum separately.
+    {
```

Comment:
LGTM. This code is meant to be between `// 1.` and `// 2.`.

Reply:
^^ please fix

Reply:
Will do.

Reply:
Done in [Review part 2.](https://github.com/linera-io/linera-protocol/pull/3377/commits/c2cad694c00323576314a0d18d5d6f2b16b4bde7).